### PR TITLE
Uc 96 create modal for user in navbar

### PR DESCRIPTION
**Pull request by edzonee**
UC-96-create-modal-for-user-in-navbar
## Description & motivation
created the modal for usermodal

created the modal for usermodal

## How to test
needs UC-96-create-modal-for-user-in-navbar from UrbanCloud-alt1
run app
run build
go to search view and modal should be there

once the steps above are done, go into the search view.

## New tickets to be added

---
- [x] The ticket is done, OR it's been mentioned why it's not done above

- [x] Removed console.log in code

- [x] Pulled dev into this branch

- [x] Resolved merge conflicts

- [x] The ID of branch is of format "ABC-123"

- [x] Added a reviewer

### CODE REVIEW

File: generics/components/templates/Template_Search_View.mjs [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Organism_Navbar } from "../organisms/Organism_Navbar.mjs"
 import { Molecule_HeadingIconAndText } from "../molecules/Molecule_HeadingIconAndText.mjs";
 import { Molecule_Paginator } from "../molecules/Molecule_Paginator.mjs"
 
+import { Molecule_UserModal } from "nd_frontend/generics/components/molecules/Molecule_UserModal.mjs";
```
Comment (lmeow) [inline generics/components/templates/Template_Search_View.mjs]: Getting an error that this doesn't exist

**Review by lmeow — CHANGES_REQUESTED** [review]
Great! However, in order to test the PR we need to fix this:

**Review by lmeow — CHANGES_REQUESTED** [review]
Just a couple of places to remove comments and console.logs, and then it's good to go! 🙌 

- I also took the liberty to move the user+logout button over to the left (which wasn't specified in the ticket)

**Review by osmo-systems — APPROVED** [review]

**Review by lmeow — APPROVED** [review]
NICE! 🚀

**Pull request merged**